### PR TITLE
bug(CUST-CPC-367): Fixing Owner Table

### PR DESCRIPTION
**Pull request by CrxJr**
Fixed the table not been created for Customer Service

### CODE REVIEW

File: api-gateway/src/main/resources/static/scripts/owner-details/owner-details.controller.js [hunk 1]
```diff
@@ -8,11 +8,11 @@ angular.module('ownerDetails')
...
 
-        $http.update('api/gateway/owners/' + $stateParams.ownerId).then(function (resp) {
+        /*$http.update('api/gateway/owners/' + $stateParams.ownerId).then(function (resp) {
             self.owner = resp.data;
         });
 
         $http.delete('api/gateway/owners/' + $stateParams.ownerId).then(function (resp) {
             self.owner = resp.data;
-        });
+        });*/
```
Comment (Louispoire) [inline api-gateway/src/main/resources/static/scripts/owner-details/owner-details.controller.js:17]: I'm not sure I understand why these lines might be the ones causing the issue. We need to make sure that we still keep in order to call an update/delete HTTP request!

File: customers-service/src/main/resources/data-h2.sql [hunk 2]
```diff
@@ -0,0 +1,31 @@
+INSERT INTO types VALUES (1, 'cat');
```
Comment (Louispoire) [inline customers-service/src/main/resources/data-h2.sql:1]: Good thing you added theses files back

File: api-gateway/src/main/resources/static/scripts/owner-details/owner-details.controller.js [hunk 3]
```diff
@@ -8,11 +8,11 @@ angular.module('ownerDetails')
             self.owner = resp.data;
         });
 
-        $http.update('api/gateway/owners/' + $stateParams.ownerId).then(function (resp) {
+        /*$http.update('api/gateway/owners/' + $stateParams.ownerId).then(function (resp) {
```
Comment (Louispoire) [inline api-gateway/src/main/resources/static/scripts/owner-details/owner-details.controller.js:11]: We should create 2 new JS file, each one containing an HTTP request like update and delete

**Review by Louispoire — APPROVED** [review]
Good stuff. By looking at your code, I see there was some issue with the angular files and html. Glad that you also re-added the sql files

**Review by Anh-Toine — APPROVED** [review]
Looks great! We should be able to do curl requests now

**Pull request merged**